base/linalg/lapack.jl

```diff
@@ -721,7 +733,7 @@ for (gesvx, elty, relty) in
             work      = Array($elty, 4n)
```

Comment:
I have tried running these `Complex` methods with `2n` instead of `4n` and everything is peachy.